src/app/public/modules/vertical-tabset/vertical-tabset-group.component.html

```diff
@@ -11,21 +12,43 @@
     [tabIndex]="-1"
   >
     {{ groupHeading }}
-    <sky-chevron *skyThemeIf="'default'"
+    <sky-chevron
       class="sky-vertical-tabset-group-header-chevron"
       [direction]="open ? 'up' : 'down'"
       [disabled]="disabled"
       (directionChange)="toggleMenuOpen()"
       (keyup.enter)="$event.stopPropagation()"
     >
     </sky-chevron>
-    <ng-container *skyThemeIf="'modern'">
+  </div>
+  <div
+    class="sky-vertical-tabset-group-content"
+    *ngIf="open"
+    [@tabSlide]
+  >
+    <ng-content></ng-content>
+  </div>
+
+  <div
+    *skyThemeIf="'modern'"
```

Comment:
The `*skyThemeIf` should go on the same line as the `<div`

Reply:
Addressed in recent commit. https://github.com/blackbaud/skyux-tabs/pull/178/commits/f23bd3b82e0f0bcaa9face16e77ee5d8720a483d